### ISSUE TITLE
Event "opened"

### ISSUE THREAD

**Issue by a3020** [euvl/vue-js-modal#15]
I'm trying to set the focus on an element once the modal is opened. However, the $refs item is undefined after the 'opened' event is emitted. I set a breakpoint and saw that the modal wasn't visible before the emit was emitted:

![event](https://cloud.githubusercontent.com/assets/1431100/25934226/e421fc8a-361c-11e7-9a81-7c20040b3cd0.PNG)

Relevant code:

```
<modal name="addContactModal" @opened="opened">
<input type="text" ref="focus"/>
```

```
opened (event) {
  this.$refs.focus.focus();
},
```

Do you have an idea?

**Comment by euvl** [comment]
Hey, thanks a lot for an issue! The event should be generated in different place tbh, in "watch" section. Will try to fix it asap.

**Comment by rmartins90** [comment]
+1

**Comment by saeedvaziry** [comment]
I still didn't get the point...
can any one tell me how to do this?

**Comment by euvl** [comment]
Its not published, didnt have time to rewier pr

**Comment by saeedvaziry** [comment]
@euvl when you can publish?

**Comment by euvl** [comment]
Whenever I will have time to do tests

**Comment by euvl** [comment]
Published in `+ vue-js-modal@1.3.2`
